File: Java/URI 1012.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
      Scanner sc = new Scanner(System.in);
      double A,B,C,T,CR,TP,Q,R;
      
      A = sc.nextDouble();
      B = sc.nextDouble();
      C = sc.nextDouble();

      T = ((A * C)/2);
      CR = (3.14159 * (C * C));
      TP = (((A + B)*C)/2);
      Q = B * B;
      R = A * B;

      System.out.printf("TRIANGULO: %.3f%n", T);
	  System.out.printf("CIRCULO: %.3f%n", CR);
	  System.out.printf("TRAPEZIO: %.3f%n", TP);
	  System.out.printf("QUADRADO: %.3f%n", Q);
	  System.out.printf("RETANGULO: %.3f%n", R);
      sc.close();
  }
}

File: C/URI 1012.c
#include <stdio.h>
int main(void) {
  double A,B,C;
  double TRIANGULO, CIRCULO, TRAPEZIO, QUADRADO, RETANGULO;
  scanf("%lf %lf %lf",&A,&B,&C);
  TRIANGULO = (A * C)/2.0;
  CIRCULO = 3.14159 * (C * C);
  TRAPEZIO = ((A + B) * C)/2.0;
  QUADRADO = B * B;
  RETANGULO = A * B;
  printf("TRIANGULO: %.3lf\n",TRIANGULO);
  printf("CIRCULO: %.3lf\n",CIRCULO);
  printf("TRAPEZIO: %.3lf\n",TRAPEZIO);
  printf("QUADRADO: %.3lf\n",QUADRADO);
  printf("RETANGULO: %.3lf\n",RETANGULO);
  return 0;
}